src/main/java/org/mockito/internal/creation/settings/CreationSettings.java

```diff
@@ -104,6 +106,10 @@ public SerializableMode getSerializableMode() {
         return invocationListeners;
     }
 
+    public List<StubbingLookUpListener> getStubbingLookUpListeners() {
```

Comment:
@szczepiq 
`Mockito.framework()` has `add`/`removeListener` to handle all types of listeners. What do you think about a unification of API listener concepts here?  I mean `CreationSettings.add/removeListener` instead of `CreationSettings.getStubbingLookUpListeners().add()` /` remove()` / `CreationSettings.getInvocationListeners().add()` /` remove()`.

This would make the API as a whole more clear and avoids different patterns for the same concept.